go.sum

```diff
@@ -3,6 +3,8 @@ cloud.google.com/go v0.31.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMT
 cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
 cloud.google.com/go v0.37.1/go.mod h1:SAbnLi6YTSPKSI0dTUEOVLCkyPfKXK8n4ibqiMoj4ok=
 contrib.go.opencensus.io/exporter/ocagent v0.4.9/go.mod h1:ueLzZcP7LPhPulEBukGn4aLh7Mx9YJwpVJ9nL2FYltw=
+git.apache.org/thrift.git v0.0.0-20180902110319-2566ecd5d999/go.mod h1:fPE2ZNJGynbRyZ4dJvy6G277gSllfV2HJqblrnkyeyg=
```

Comment:
I wonder how this appeared in the last commit

Reply:
i pulled in master, and it was there. Should i remove this ?

Reply:
Did you pull it in on the same commit? Usually there is an intermediate merge commit. Also if that same change is on master, I think it would either say there is a conflict, or not show up in the diff. Not entirely sure though.

I'd say remove it if CI passes without it. I don't see a reason for a dep update in this PR.

Reply:
yes got it